child_compassion/models/child_compassion.py

```diff
@@ -537,6 +537,30 @@ def fetch_translations(self):
         self._fetch_translations(self.env.ref("child_compassion.beneficiaries_details"))
         return self.edit_translations()
 
+    @api.model
+    def update_all_children_description(self):
+        children = self.env["compassion.child"].sudo().search([
+            ("state", "not in", ["F", "R"])
+        ])
+        infos = ""
+        for child in children:
+            child.with_delay(priority=50).update_child_descriptions()
+
+    def update_child_descriptions(self):
+        descriptions = (
+            self.env["compassion.child.description"]
+            .sudo()
+            .search([("child_id", "=", self.id)], order="write_date desc", limit=1)
+        )
+        if len(descriptions) > 0:
+            description = descriptions[0]
+            description._generate_all_translations()
+            return ""
+        return "Error with %s (%s) : description not found" % (
+            self.name if self.name else "",
+            self.id,
+        )
```

Comment:
compassion.child.description is a wizard. The table is therefore most of the time empty. The process for generating a description is simply to create one record in this table. I know it's a bit an akward design but this is a legacy functionality we may once rethink better.
```suggestion
        self.ensure_one()
        return self.env["compassion.child.description"].create({
            "child_id": self.id,
        })
```